Rakefile

```diff
@@ -18,7 +18,7 @@ task :validate do
 end
 
 task :acceptance do
-  sh "puppet module build"
+  sh "gem build puppet-strings.gemspec"
```

Comment:
@zreichert we should move rototiller to use this method of install, instead of scp the source